Vagrantfile

```diff
@@ -10,8 +10,11 @@ apt-get install -y ca-certificates apt-transport-https
 
 echo >/etc/apt/sources.list.d/weasyl.list \
     'deb http://apt.weasyldev.com/repos/apt/debian jessie main'
+echo >/etc/apt/sources.list.d/postgresql.list \
+    'deb http://apt.postgresql.org/pub/repos/apt/ trusty-pgdg main 9.5'
```

Comment:
Any reason we’re not using HTTPS for apt.weasyldev.com, then? (I’m not sure it matters, since everything’s signed anyway.)